plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestingMemSqlServer.java

```diff
@@ -44,6 +44,12 @@ public TestingMemSqlServer(String dockerImageName)
     {
         super(DockerImageName.parse(dockerImageName));
         addEnv("ROOT_PASSWORD", "memsql_root_password");
+        withCommand("sh", "-xeuc",
+                "/startup " +
+                // Lower the size of pre-allocated log files to 1MB (minimum allowed) to reduce disk footprint
+                "&& memsql-admin update-config --yes --all --set-global --key \"log_file_size_partitions\" --value \"1048576\" " +
+                "&& memsql-admin update-config --yes --all --set-global --key \"log_file_size_ref_dbs\" --value \"1048576\" " +
+                "&& /startup");
```

Comment:
why `/startup` is invoked twice?

Reply:
Removing the `START_AFTER_INIT` env variable means that `startup` just sets up the cluster on first run. Re-running it actually starts the cluster (it detects that setup is already done).

It needs to be done this way because the configs I changed requires nodes to be restarted.